scripts/deploy-introspection-schema.sh

```diff
@@ -0,0 +1,24 @@
+# Taking a compressed introspection schema file and uploading it to S3
+# This is specifically for s3://graphiq.medplum.com/schema/
+
+version=$1
+client_id=$2
+client_secret=$3
+cors_file=$4
```

Comment:
What goes into `cors_file`?  If it's constant, can we include it here in the bash script?